admin/commands/management/remove.go

```diff
@@ -55,6 +98,14 @@ func (cmd *removeMySQLCommand) Run() (commands.Result, error) {
 	}
 	_, err := client.Default.Service.RemoveService(params)
 	if err != nil {
+		e, ok := err.(commands.ErrorResponse)
+		if !ok {
+			return nil, err
+		}
+		errResponse := commands.GetError(e)
+		if autoLookup && errResponse.Code < 500 {
```

Comment:
We should know the exact code number for this response, so let's use it.

Reply:
Do we really need this code because we return an error above

Reply:
not anymore, we can completely remove this part of the code